### DISCUSSION TITLE
Build workflow targets the right branch but wrong repo?

### DISCUSSION THREAD

**Discussion by tjvezina** [community/community#35839] [Actions]
I'm setting up GitHub Actions for the first time (in C# repo [decline-cookies/anvil-csharp-core](https://github.com/decline-cookies/anvil-csharp-core)) and added two workflows ([see here](https://github.com/decline-cookies/anvil-csharp-core/tree/main/.github/workflows)), one for .NET builds and one for mono builds, which are triggered by PR's.

The .NET workflow works as expected, running against `<fork>:<branch_name>`, but the mono build gives "Startup failure" and appears to be targetting simply `<branch_name>` (missing fork name), which doesn't exist in the main repo. The error given is:
```
Error: .github#L1
microsoft/setup-msbuild@v1.1 is not allowed to be used in decline-cookies/anvil-csharp-core. Actions in this workflow must be: within a repository owned by decline-cookies or created by GitHub.
```
...which makes sense, because it's targeting a branch that doesn't exist. Can anyone help determine why this is happening?

**Comment by ruhanpandit** [comment]
do you have git installed properly?

**Reply by 772library772** [reply]
i think thats the problem

**Reply by tjvezina** [reply]
On my local machine? Yes, I've been using git/GitHub for years, no other git issues - how would that affect a build action running in GitHub when a PR is opened?
